apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -168,6 +168,11 @@ type CloneSetStatus struct {
 	// indicated by updateRevision and have a Ready Condition.
 	UpdatedReadyReplicas int32 `json:"updatedReadyReplicas"`
 
+	// ExpectedPartitionReplicas is the number of Pods that should be partitioned by CloneSet controller. This field
+	// is designed to let users know the absolute number of Partition calculated by CloneSet controller if they use
+	// percentage as their Partition.
+	ExpectedPartitionReplicas int32 `json:"expectedPartitionReplicas,omitempty"`
```

Comment:
plz rename the field to expectUpdatedReplicas which should be more understandable for users